src/pages/Environments.js

```diff
@@ -321,6 +320,11 @@ const Environments = () => {
     })
   }
 
+  const multipleDisksError = (disks, appType) => {
+    return !!appType ? workspaceHasMultipleDisks(disks, appType) : _.remove(disk => getDiskAppType(disk) !== appType || disk.status === 'Deleting',
```

Comment:
I think a comment would be good to explain that we expect appType to be `undefined` in the case of runtimes (and therefore the second part of the conditional is filtering out disks that are app disks instead of runtime disks).

Reply:
i'll add one